src/pages/Login.vue

```diff
@@ -22,10 +22,18 @@
             bg-color="white"
             filled
             class="q-px-sm form-input col-xs-12 col-md-12"
-            type="password"
+            :type="showPassword ? 'password' : 'text'"
```

Comment:
how did this end up here ? Oo

Reply:
I see above you merged development and the password toggle branch, why did you do that ?